cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -402,9 +405,33 @@ Depending on your installation, you may want to set these properties:
       <description>Enable Explore functionality</description>
     </property>
 
-  **Note:** This feature cannot be used unless the cluster has a correct version of Hive installed.
-  See :ref:`Hadoop/HBase Environment <install-hadoop-hbase>`. This feature is currently 
-  not supported on secure Hadoop clusters.
+  This feature cannot be used unless the cluster has a correct version of Hive installed.
+  See :ref:`Hadoop/HBase Environment <install-hadoop-hbase>`. If you have enabled the CDAP Explore Service on
+  a secure Hadoop cluster, you will need to add a few additional Hadoop configuration settings.
+
+  In order to allow the Hive Metastore to run operations as the cdap user,
+  add the following settings to the Hadoop ``core-site.xml`` file::
+
+    <property>
+      <name>hadoop.proxyuser.hive.groups</name>
+      <value>cdap,hadoop,hive</value>
+    </property>
+    <property>
+      <name>hadoop.proxyuser.hive.hosts</name>
+      <value>*</value>
+    </property>
+
+  If you are running CDAP as a user other than ``cdap``, replace ``cdap`` with the correct user.
+
+  **Note:** Some versions of Hive contain a bug that may prevent the CDAP Explore Service from starting
+  up. See `CDAP-1865 <https://issues.cask.co/browse/CDAP-1865>`__ for more information about the issue.
+  If the CDAP Explore Service fails to start and you see a ``javax.jdo.JDODataStoreException: Communications link failure``,
```

Comment:
Can you take out the first comma after "failure"? I think just one comma after "log" is enough...do you?